packages/ui/core-components/src/lib/atoms/inputs/slider/Slider.svelte

```diff
@@ -25,35 +26,11 @@
 	/** @type {number|undefined} */
 	export let min = 0;
 
-	if (min) {
-		// if min was user-supplied
-		min = Number(min);
-		if (isNaN(min)) {
-			// input must be a number
-			throw Error('min must be a number');
-		}
-	} else {
-		min = 0;
-	}
-
 	/** @type {number} */
 	export let max = 100;
 
-	if (max) {
-		// if min was user-supplied
-		max = Number(max);
-		if (isNaN(max)) {
-			// input must be a number
-			throw Error('max must be a number');
-		} else if (max < min) {
-			throw Error('max cannot be less than min');
-		}
-	} else {
-		max = 100;
-	}
-
 	/** @type {number} */
-	export let step;
+	export let step = undefined;
```

Comment:
We should default `step` to `1`

```suggestion
	export let step = 1;
```